app/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -28,20 +28,32 @@ import TableOverview from "./TableOverview";
 type SummaryProps = {
   experiment: getExperiment_experimentBySlug;
   refetch?: () => void;
-} & Partial<React.ComponentProps<typeof ChangeApprovalOperations>>; // TODO EXP-1143: temporary page-level props, should be replaced by API data for experiment & current user
+};
 
 const Summary = ({ experiment, refetch }: SummaryProps) => {
   const status = getStatus(experiment);
 
   const {
     isLoading,
     submitError,
-    callbacks: [onConfirmEndClicked],
-  } = useChangeOperationMutation(experiment, refetch, {
-    publishStatus: NimbusExperimentPublishStatus.REVIEW,
-    statusNext: NimbusExperimentStatus.COMPLETE,
-    changelogMessage: CHANGELOG_MESSAGES.REQUESTED_REVIEW_END,
-  });
+    callbacks: [onConfirmEndClicked, onConfirmEndEnrollmentClicked],
+  } = useChangeOperationMutation(
+    experiment,
+    refetch,
+    {
+      publishStatus: NimbusExperimentPublishStatus.REVIEW,
+      status: NimbusExperimentStatus.LIVE,
+      statusNext: NimbusExperimentStatus.COMPLETE,
+      changelogMessage: CHANGELOG_MESSAGES.REQUESTED_REVIEW_END,
+    },
+    {
+      publishStatus: NimbusExperimentPublishStatus.REVIEW,
+      status: NimbusExperimentStatus.LIVE,
+      statusNext: NimbusExperimentStatus.LIVE,
+      // TODO: EXP-1325 Need to set something else here for end enrollment in particular?
```

Comment:
I'm guessing this ties in with [this TODO](https://github.com/mozilla/experimenter/pull/5852/files#r666457588) where at the moment I'm not sure what else we need